packages/map/src/vaadin-map.js

```diff
@@ -52,7 +53,7 @@ function isEnabled() {
  * @mixes ThemableMixin
  * @mixes ElementMixin
  */
-class Map extends ElementMixin(ThemableMixin(PolymerElement)) {
+class Map extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))) {
```

Comment:
nit: let's add `@mixes ResizeMixin` above for generating API docs.

Reply:
Done